### PR TITLE
notification script for mac os

### PR DESCRIPTION
**Pull request by IshaanRawat**
Hi,

I have added a file for mac os which has the updated command for notifying on mac os.

### CODE REVIEW

File: scripts/macos/cowin-cli-notify-book.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Anoop S
+# shell script to notify and book using cowin-cli
+# Install notify-send for notifications
```
Comment (anoop142) [inline scripts/macos/cowin-cli-notify-book.sh]: Remove this comment 
 Install notify-send for notifications

**Review by anoop142 — CHANGES_REQUESTED** [review]
edit  notify-send comment out

**Comment by IshaanRawat** [comment]
Ok.

**Comment by IshaanRawat** [comment]
> edit notify-send comment out

Done.

**Pull request merged**